### PR TITLE
Implement MVP1 - Nav. Remove shadows (#189)

### PR DESCRIPTION
**Pull request by MightyWizard83**
I don't know if i've followed the correct procedure (probably not)

but I'd like to contribute (even though i'm mainly a backend developer, not a frontender)

### CODE REVIEW

**Comment by bryansamuel1** [comment]
@MightyWizard83 can you switch from master to develop as we don't directly merge into master.
Also check that you have rebased against the latest develop branch if you have not done so already.

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
```
Comment (bryansamuel1) [inline .github/FUNDING.yml:1]: @NikoGrano what are these?

File: .gitlab/auto-deploy-values.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 deploymentApiVersion: apps/v1
-replicaCount: 3
+replicaCount: 1
 strategyType: RollingUpdate
```
Comment (bryansamuel1) [inline .gitlab/auto-deploy-values.yaml:3]: @NikoGrano what are these?

File: src/components/navigation/TheNavLarge.vue [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <template>
```
Comment (bryansamuel1) [inline src/components/navigation/TheNavLarge.vue:1]: got a screenshot of what it looks like now @MightyWizard83 ?

Comment (MightyWizard83) [reply src/components/navigation/TheNavLarge.vue:1]: @bryansamuel1 This is the LARGE:
image

Comment (MightyWizard83) [reply src/components/navigation/TheNavLarge.vue:1]: This is the SMALL:
image

Comment (MightyWizard83) [reply src/components/navigation/TheNavLarge.vue:1]: P.S. I've branched from MASTER (my bad)

I've never rebased after pushing to the origin. I'm not sure how to do it.

If you want I can reimplement it and make another pull request (it's only a couple of lines of code)

File: .github/FUNDING.yml (hunk 1 above)
Comment (NikoGrano) [reply .github/FUNDING.yml:1]: Adds "Sponsor" Button up there.

File: .gitlab/auto-deploy-values.yaml (hunk 2 above)
Comment (NikoGrano) [reply .gitlab/auto-deploy-values.yaml:3]: Kubernetes

**Pull request closed without merging**